src/components/cron.ts

```diff
@@ -42,6 +42,9 @@ export const createOfficeStatusCron = (): CronJob =>
         //discord channel names don't accept :emoji: so we have to use actual unicode
         (response['status'] == 1 ? '✅' : response['status'] == 0 ? '❌' : '❓');
       (messageChannel as TextChannel).setName(curName);
+      const time = Math.floor(response['time']);
+      const topic = `Last Updated At <t:${time}:F> for you (<t:${time}:R>)`;
```

Comment:
Just a super small nitpick, I think it would look nicer if the casing is "Last updated at" instead.